packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -537,6 +537,7 @@ const messages = {
       files: 'Files',
       embedlink: 'Embedded link',
       concept: 'Concepts',
+      gloss: 'Glossary',
```

Comment:
Trur nesten vi må endre denne også.
```suggestion
      gloss: 'Glosses',
```